### PR TITLE
feat(Create Hero)

### PR DESCRIPTION
**Pull request by MLKendall**
Not re-based to gatsby-version - looking for feedback first.

### CODE REVIEW

File: src/html.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            sizes="16x16"
+            type="image/png"
+          />
+          <link
+            color="#5bbad5"
+            href="/img/safari-pinned-tab.svg"
+            rel="mask-icon"
+          />
+          <link
+            href="https://fonts.googleapis.com/css?family=Roboto:100, 100i, 300,300i,400,500"
```
Comment (csteinert) [inline src/html.jsx]: The spaces here are breaking the URL. Remove them and the Hero Tagline will have the correct weight. ✌️

**Comment by csteinert** [comment]
I really like, in the mocks, how the tagline is vertically equidistant from the logo and the satellite image. In the code, though, the top margin on the satellite image is putting more space between the two. Not sure if that was intentional, but subjectively I like it better with equal spacing.

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "@types/node": "^8.0.54",
     "@types/react": "^16.0.26",
     "@types/react-dom": "^16.0.3",
+    "editorconfig": "^0.15.0",
```
Comment (mewdriller) [inline package.json]: Was this needed?

File: src/components/atoms/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  grid-template-rows: 100vh;
+
+  ${mediaQuery.small`
+    grid-template-columns: 2em 1fr 2em;
+    grid-template-rows: 1958px;
+  `};
+
+  ${mediaQuery.medium`
+  grid-template-columns: 100px 1fr 100px;
+`};
```
Comment (mewdriller) [inline src/components/atoms/Main.jsx]: Align media-query indentation.

File: src/components/molecules/DescriptiveList.jsx [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
-
-  ${mediaQuery.medium`
-    grid-template-columns: 1fr 1fr 1fr;
-  `};
-`;
-
-DescriptiveList.Item = DescriptiveListItem;
-
-export default DescriptiveList;
+// import PropTypes from 'prop-types';
```
Comment (mewdriller) [inline src/components/molecules/DescriptiveList.jsx]: If we're no longer using this (commenting it out), we should remove it from the project.

File: src/components/molecules/GlobalNavigation.jsx [hunk 5]
```diff
@@ -11,7 +11,207 @@ class GlobalNavigation extends React.Component {
       <nav>
         <ul>
           <li>
-            <Link to="/">Base Two</Link>
+            <Link to="/" aria-label="Base Two Home">
+              <svg
```
Comment (mewdriller) [inline src/components/molecules/GlobalNavigation.jsx]: Encapsulate this as an atom component for reuse elsewhere.

File: src/components/organisms/Hero.jsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    margin-top: 12rem;
+  `};
+
+  ${mediaQuery.large`
+    margin-top: 17rem;
+  `};
+
+  ${mediaQuery.xlarge`
+  margin-top: 18rem;
+`};
```
Comment (mewdriller) [inline src/components/organisms/Hero.jsx]: Align media-query indentation.

File: src/components/organisms/Hero.jsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  width: 80%;
+  max-width: 750px;
+
+  ${mediaQuery.small`
+  width: 65%;
+`};
+`;
+
+const HeroHeading = styled(H1)`
+  //This is to prevent the extra div around the image in Logo from addng content space at the top
```
Comment (mewdriller) [inline src/components/organisms/Hero.jsx]: Remove comment.

File: src/html.jsx [hunk 8]
```diff
@@ -86,7 +86,7 @@ class Html extends React.Component {
             rel="mask-icon"
           />
           <link
-            href="https://fonts.googleapis.com/css?family=Roboto:400,500"
+            href="https://fonts.googleapis.com/css?family=Roboto:100,100i,400,500"
```
Comment (mewdriller) [inline src/html.jsx]: We'll want to consider the request weight of these at some point for performance.

File: static/img/b2CirclesOnly.svg [hunk 9]
```diff
@@ -0,0 +1,95 @@
+<svg width="102" height="48" viewBox="0 0 102 48" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (mewdriller) [inline static/img/b2CirclesOnly.svg]: Encapsulate these two SVG files as React components so other places can use them more easily.  You can _probably_ import the `**/*.svg` path and get this to come in as elements (assuming Gatsby setup that webpack loader for us). If that doesn't work, there's almost certainly a Gatsby plugin for that or we can manually update the webpack loaders.

Comment (mewdriller) [inline static/img/b2CirclesOnly.svg]: Have these been through something like `svgo`?

**Pull request merged**